demo/video-cast.js

```diff
@@ -21,9 +21,11 @@ import ChromecastControls from '../src/chromecast-controls.js';
 
     document.getElementById('video-button-start').addEventListener('click', playVideo);
     document.getElementById('video-button-stop').addEventListener('click', stopCastVideo);
-
-    player.setupChromecast('#cast-wrapper', chromecastReceiverAppId);
-    document.getElementById('cast-wrapper').style.display = 'unset';
+    player.setupChromecast('#cast-wrapper', chromecastReceiverAppId)
+        .then(() => {
+            const controls = new ChromecastControls(player.getCastPlayer(), player.getCastPlayerController());
+            document.getElementById('cast-wrapper').style.display = 'unset';
```

Comment:
can the cast-wrapper style.display be unset in our library?

Reply:
It's not part of the library but only our example. Other devs may not even hide it etc.